src/main.rs

```diff
@@ -34,11 +33,11 @@ pub fn main() {
         .run();
 }
 
-fn render_origin(mut gizmos: Gizmos) {
-    gizmos.line(Vec3::ZERO, Vec3::X, Color::RED);
-    gizmos.line(Vec3::ZERO, Vec3::Y, Color::GREEN);
-    gizmos.line(Vec3::ZERO, Vec3::Z, Color::BLUE);
-}
+// fn render_origin(mut gizmos: Gizmos) {
+//     gizmos.line(Vec3::ZERO, Vec3::X, Color::RED);
+//     gizmos.line(Vec3::ZERO, Vec3::Y, Color::GREEN);
+//     gizmos.line(Vec3::ZERO, Vec3::Z, Color::BLUE);
+// }
```

Comment:
Commenting code in and out like this results in noise in your commit messages and makes it harder for me to see what has changed. Commits should always contain as few changes as possible.